### DISCUSSION TITLE
Is it possible to access the entry object when using x-intersect?

### DISCUSSION THREAD

**Discussion by partcoffee** [alpinejs/alpine#3613] [1. Help] [answered]
I want to use `x-intersect`, but also access `entry.boundingClientRect`? When using `x-on`, the `$event` object is exposed, but I'm not seeing anything documented for `x-intersect`.

**Comment by ekwoka** [comment, marked as answer]
It does not expose any extra values.

For this specific case you can just use `$el.getBoundingClientRect()`

**Reply by partcoffee** [reply]
Gotcha, thanks for the clarification. I was trying to prevent reflows, so `getBoundingClientRect()` is unfortunately not the way to go.

**Reply by ekwoka** [reply]
How so? It's the same thing, no?

The `entry.boundingClientRect` is literally just the result from calling `element.getBoundingClientRect()`. So there shouldn't be any differences as far as reflow, etc.

**Reply by partcoffee** [reply]
I believe `entry.boundingClientRect` operates with the same algorithm, but reading it shouldn't [trigger a reflow](https://gist.github.com/paulirish/5d52fb081b3570c81e3a). I have yet to test this properly, so I'm happy to be wrong on this.

**Reply by ekwoka** [reply]
Based on that article, the issue is more when the layout has actually changed.

If nothing has changed it doesn't have much cost. The issue arrises when the layout changes between calls to such apis.
